releasenotes/notes/target-aware-layout-routing-2b39bd87a9f928e7.yaml

```diff
@@ -0,0 +1,34 @@
+---
+features:
+  - |
+    The following layout and routing transpiler passes from the
+    :mod:`.qiskit.transpiler.passes` modules have a new keyword argument,
+    ``target`` which takes in a :class:`~.Target` object which is used to model
+    the constraints of a target backend. If the ``target`` keyword argument is
```

Comment:
This isn't quite true any more.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9263/commits/4aa28dc8ff788fca5b4ed8e86a70a766bfb2cbbc